datastore/src/test/java/io/spine/server/storage/datastore/tenant/NamespaceIndexTest.java

```diff
@@ -218,49 +215,49 @@ private static void testSynchronizeAccessMethods() throws InterruptedException {
         keys.add(key("Vtenant3"));
         Collection<TenantId> initialTenantIds =
                 keys.stream()
-                    .map(key -> TenantId.newBuilder()
-                                        .setValue(key.getName()
-                                                     .substring(1))
-                                        .vBuild())
-                    .collect(toList());
+                        .map(key -> TenantId.newBuilder()
+                                .setValue(key.getName()
+                                             .substring(1))
+                                .vBuild())
+                        .collect(toList());
 
         NamespaceIndex.NamespaceQuery namespaceQuery = keys::iterator;
         // The tested object
-        NamespaceIndex namespaceIndex = nsIndexFor(namespaceQuery);
+        var namespaceIndex = nsIndexFor(namespaceQuery);
 
         // The test flow
         Runnable flow = () -> {
             // Initial value check
-            Set<TenantId> initialIdsActual = namespaceIndex.all(); // sync
+            var initialIdsActual = namespaceIndex.all(); // sync
             // The keep may already be called
             assertTrue(initialIdsActual.size() >= initialTenantIds.size());
             assertThat(initialIdsActual).containsAtLeastElementsIn(initialTenantIds);
 
             // Add new element
-            InternetDomain domain = InternetDomain
+            var domain = InternetDomain
                     .newBuilder()
                     .setValue("my.tenant.com")
                     .vBuild();
-            TenantId newTenantId = TenantId
+            var newTenantId = TenantId
                     .newBuilder()
                     .setDomain(domain)
                     .vBuild();
             namespaceIndex.keep(newTenantId); // sync
 
             // Check new value added
-            boolean success = namespaceIndex.contains(Namespace.of(newTenantId,    // sync
-                                                                   true));
+            var success = namespaceIndex.contains(Namespace.of(newTenantId,    // sync
+                                                               true));
             assertTrue(success);
 
             // Check returned set has newly added element
-            Set<TenantId> updatedIds = namespaceIndex.all(); // sync
+            var updatedIds = namespaceIndex.all(); // sync
             assertEquals(updatedIds.size(), initialTenantIds.size() + 1);
             assertThat(updatedIds).contains(newTenantId);
         };
 
         // Test execution threads
-        Thread firstThread = new Thread(flow);
-        Thread secondThread = new Thread(flow);
+        var firstThread = new Thread(flow);
+        var secondThread = new Thread(flow);
 
         // Collect thread failures
         Map<Thread, Throwable> threadFailures = new HashMap<>(2);
```

Comment:
We should consider
```java
var threadFailures = new HashMap<Thread, Throwable>(2);
```
Seems a bit more concise.
However, the next line with the method reference cast to a functional interface looks just right without the `var` to me.